### PR TITLE
Updated ARC RO-Crate Profile Description

### PR DESCRIPTION
**Pull request by floWetzels**
The description of the ARC RO-Crate profile now uses the new ISA RO-Crate profile based on the new/adapted bioschemas types LabProcess and LabProtocol.

### CODE REVIEW

File: ARC specification.md [hunk 1]
```diff
@@ -493,14 +493,15 @@ As the ARC might be used by different persons and in different workflow contexts
 An ARC can be augmented into an RO Crate by placing a metadata file `ro-crate-metadata.json` into the top-level ARC folder, which must conform to the [RO Crate specification](https://www.researchobject.org/ro-crate/1.1/).
 The ARC root folder is then simultaneously the RO Crate Root and represents an ISA investigation.
```
Comment (HLWeil) [inline ARC specification.md:494]: Ofc not changed in this PR, but.. 

Is this technically even possible? Or do we have to put the Investigation also as a property into the RO-Crate Root?

File: ARC specification.md [hunk 2]
```diff
@@ -493,14 +493,15 @@ As the ARC might be used by different persons and in different workflow contexts
...
   - The root data entity description are taken from the "Investigation Description" term in `isa.investigation.xlsx`.
   - The root data entity authors are taken from the "Investigation Contacts" in `isa.investigation.xlsx`:
   - The root data entity citations are taken from the "Investigation Publications" section in `isa.investigation.xlsx`.
 - For each assay and study linked from `isa.investigation.xlsx`, one dataset entity is provided in `ro-crate-metadata.json`. The Dataset id corresponds to the relative path of the assay ISA file under `assays/`, e.g. "sample-data/isa.assay.xlsx". Other metadata is taken from the corresponding terms in the corresponding `isa.assay.xlsx` or `isa.study.xlsx`.
 - The root data entity is connected to each assay and study through the `hasPart` Property.
+- The assay and study entities follow the [ISA Assay Profile](https://github.com/nfdi4plants/isa-ro-crate-profile/blob/main/profile/isa_ro_crate.md) or the [ISA Study Profile](https://github.com/nfdi4plants/isa-ro-crate-profile/blob/main/profile/isa_ro_crate.md), respectively.
```
Comment (HLWeil) [inline ARC specification.md:505]: Maybe add these specifications directly to the bits targeting assay and study respectively. Just like the investigation part a few rows above.

File: ARC specification.md (hunk 1 above)
Comment (floWetzels) [reply ARC specification.md:494]: The root data entity should be the root directory, which corresponds to the ISA investigation. Thus, I would keep it as is.

File: ARC specification.md (hunk 2 above)
Comment (floWetzels) [reply ARC specification.md:505]: I think that these two bullet points say different things. First, we say that there are data entities for assay and studies (their corresponding directories). Then, we say that these are datasets following the corresponding profiles.

**Review by HLWeil — APPROVED** [review]

**Pull request merged**